tools/wpt/browser.py

```diff
@@ -17,6 +17,10 @@
 
 uname = platform.uname()
 
+# the rootUrl for the firefox-ci deployment of Taskcluster
+# (after November 9, https://firefox-ci-tc.services.mozilla.com/)
```

Comment:
Does this already work or do we actually need to remember to switch the values after Firefox changes?

Reply:
It doesn't work yet because there are no browsers built in that deployment yet, so it will need to switch on or after November 9.  After November 9, the old URLs will work, but won't be updated with new builds.  So, the timing doesn't need to be precise - Monday would be fine.